geode-core/src/main/java/org/apache/geode/management/CacheServerMXBean.java

```diff
@@ -352,44 +352,44 @@
    * 
    * @param clientId ID of the client for which to retrieve information.
    */
-  public ClientHealthStatus showClientStats(String clientId) throws Exception;
+  ClientHealthStatus showClientStats(String clientId) throws Exception;
 
   /**
    * Returns the number of clients who have existing subscriptions.
    */
-  public int getNumSubscriptions();
+  int getNumSubscriptions();
 
   /**
    * Returns health and statistic information for all clients. Some of the information (CPUs,
    * NumOfCacheListenerCalls, NumOfGets,NumOfMisses, NumOfPuts,NumOfThreads, ProcessCpuTime) only
    * available for clients which have set a "StatisticsInterval".
    */
-  public ClientHealthStatus[] showAllClientStats() throws Exception;
+  ClientHealthStatus[] showAllClientStats() throws Exception;
 
   /**
    * Shows a list of client with their queue statistics. Client queue statistics shown in this
    * method are the following
    * 
-   * eventsEnqued,eventsRemoved , eventsConflated ,markerEventsConflated , eventsExpired,
+   * eventsEnqueued,eventsRemoved , eventsConflated ,markerEventsConflated , eventsExpired,
```

Comment:
(Nit) This (and the following javadoc comment) are kinda ugly with the spacing around the commas. While you're fixing the spelling why not also fix the rest.